storagenode/piecestore/verification_test.go

```diff
@@ -290,10 +291,10 @@ func TestOrderLimitGetValidation(t *testing.T) {
 		require.NoError(t, err)
 
 		buffer, readErr := ioutil.ReadAll(downloader)
```

Comment:
:+1:

Reply:
Although, it also might be a bug in limiting itself, if we do 10 concurrent uploads all with 30kib, and the limit is 30kib only one of those should succeed.

Reply:
Even with my original fix in place i think that would be possible if you got lucky with timing.

Reply:
I had considered that it might have been done to save time but err'd on the side of correctness. When you brought up that it was reduce latency with the uplink, I thought to myself that there were probably other ways to defeat the limiting and the amount would (hopefully) be small so I was ok just patching the test.

Reply:
Yeah, let's patch the test and add an issue to backlog about concurrent uploads not respecting bandwidth limit.

Reply:
@egonelbre if this looks good to you as-is, can you approve it?

Reply:
PS: doesn't `TestOrderLimitPutValidation` require a similar fix?

Reply:
Ah yes, it will, good catch. @azdagron can you add a fix to that test as well? (Either in this PR or a separate one?

Reply:
I don't think `TestOrderLimitPutValidation` will hit this. Those tests are only performing a single operation, so they don't have to worry about the state change (or timing of that change) from a previous operation.

Reply:
You should be able to verify this by adding  a time.Sleep(5*time.Second) in the test inside the defer.

Reply:
Confirmed that `TestOrderLimitPutValidation` does not have this problem.